README.md

```diff
@@ -42,6 +42,7 @@ Usage: scanpy-cli [OPTIONS] COMMAND [ARGS]...
 Options:
   --debug              Print debug information
   --verbosity INTEGER  Set scanpy verbosity
+  --njobs INTEGER      Set scanpy default number of jobs/CPUs, defaults to all available
```

Comment:
I would suggest that you keep the default as it was (1 job), then on updates of older scripts, you don't need to change anything (but you can always set the number of jobs explicitly as pursued). Probably also using all available CPUs by default might lead (when forgetting to set this) to some undesirable behaviour (abuse node beyond resources request, use more memory than requested and then get killed due to OOM, etc).

Reply:
Thanks, Pablo. Very much appreciate this commnent. We were also planning on setting the number of jobs explicitly where applicable while the pipeline is still using an older version of `scanpy-scripts`.  Do you think perhaps that having this option in `scanpy-cli` might be useful to the public, with default value set to 1?